docs/api/v1/openapi.yml

```diff
@@ -1022,7 +1020,12 @@ paths:
       summary: Delete a project
       description: Delete a project from your instance.
       parameters:
-        - $ref: '#/components/parameters/projectId'
+        - in: path
```

Comment:
Just out of curiosity: this seems to render the same as before. Is defining these inline preferred now?